docs/authentication/session-management-overview.mdx

```diff
@@ -0,0 +1,101 @@
+
+When you request an [access token](https://datatracker.ietf.org/doc/html/rfc6749#section-1.4), it is possible to receive an additional token called a [refresh token](https://datatracker.ietf.org/doc/html/rfc6749#section-1.5) in the response. The refresh token can be used to request additional access tokens with identical or narrower scope.
+
+## How to add session management to my application
+
+There are a few functions of tokens that you should be aware of.
+
+- [Create a Token](/docs/next/create-api-token) (Pre-Requisite)
+- [Store Tokens](#how-to-store-tokens)
+- [Introspect a Token](/docs/next/validate-access-tokens)
+- [Revoke a Token](/docs/next/revoke-access-tokens)
+- [Retrieve Active Tokens](/docs/next/list-access-tokens)
+
+## How to store tokens
+
+Applications may utilize cookies, local storage or other strategies to associate session with the end-user’s application. Beyond Identity doesn’t provide an API/SDK for this purpose; it is up to you to decide how to handle the end-user’s session. One option is to store the JWT received from Beyond Identity in the end user’s application. Another solution is to store the JWT received from Beyond Identity on the server and use a cookie. This is a common solution offered by many web frameworks and is more secure as the JWT is more sensitive.
+
+### JWT Strategy with Refresh/Access Tokens
+
+In the context of session management, the central element is the JSON Web Token (JWT), which serves as the token that encapsulates and conveys the authentication and authorization information necessary for maintaining secure user sessions.
+
+#### Access Tokens:
+
+- Access tokens play a pivotal role in this strategy. These tokens are short-lived JWTs issued upon successful user authentication.
+
+- They contain crucial user details and permissions in the form of claims.
+
+- When a user interacts with the application, they present the access token in their requests. The server validates the access token to ascertain the user's identity and determine the extent of their authorized actions within the application.
+
+#### Refresh Tokens:
+
+- Refresh tokens complement access tokens within the JWT strategy. These tokens are designed to have a longer lifespan compared to access tokens.
+
+- Their primary purpose is to securely facilitate the renewal of access tokens without necessitating frequent user logins.
+
+- In cases where an access token expires, the client can utilize the refresh token to request a new access token without needing the user's credentials again.
+
+- It's important to emphasize that refresh tokens should be stored and handled with great care, as their exposure could lead to unauthorized access. They should not be accessible by client-side scripts or prone to XSS attacks.
+
+#### Why/Why Not to Use Local Storage with JWTs:
```

Comment:
I'd rephrase the heading "Where to store tokens" as we seem to have a clear recommendation that local storage is not recommended, so "why/why not" is misleadingly neutral

Reply:
Done 👍